File: tests/js/sentry-test/fixtures/apiKey.js
export function ApiKey(params = {}) {
  return {
    allowed_origins: '',
    id: 1,
    key: 'aa624bcc12024702a202cd90be5feda0',
    label: 'Default',
    scope_list: ['project:read', 'event:read', 'team:read', 'member:read'],
    status: 0,
    ...params,
  };
}

File: packages/sentry/src/sentry/api/serializers/models/apikey.py
from sentry.api.serializers import Serializer, register
from sentry.models import ApiKey


@register(ApiKey)
class ApiKeySerializer(Serializer):
    def serialize(self, obj, attrs, user):
        return {
            "id": str(obj.id),
            "label": obj.label,
            "key": obj.key,
            "scope_list": obj.scope_list,
            "status": obj.status,
            "allowed_origins": "" if obj.allowed_origins is None else str(obj.allowed_origins),
        }
